dotfiles_template/shell/zsh/.zimrc

```diff
@@ -2,8 +2,6 @@
 zmodule zimfw/environment
 zmodule zimfw/git-info
 zmodule zimfw/input
-zmodule zsh-users/zsh-autosuggestions
-zmodule zsh-users/zsh-syntax-highlighting
 zmodule zimfw/completion
-
-zhighlighters=(main brackets)
+zmodule zsh-users/zsh-syntax-highlighting
+zmodule zsh-users/zsh-autosuggestions
```

Comment:
It's recommended to source zsh-users/zsh-syntax-highlighting after completion. Likewise, zsh-users/zsh-autosuggestions goes better after zsh-users/zsh-syntax-highlighting.